### PR TITLE
core(scoring): rebalance perf metric weightings

### PR DESCRIPTION
**Pull request by paulirish**
You can easily compare v8 with v6/7 scoring here: https://googlechrome.github.io/lighthouse/scorecalc/#FCP=3000&SI=5800&FMP=4000&TTI=7300&FCI=6500&LCP=4000&TBT=600&CLS=0.25&device=mobile&version=8&version=6&version=5

Readable summary and FAQ coming. 

For now, only internal draft at go/lh-perf-v8

### CODE REVIEW

File: lighthouse-core/report/html/renderer/performance-category-renderer.js [hunk 1]
```diff
@@ -112,11 +112,15 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
    * @return {string}
    */
   _getScoringCalculatorHref(auditRefs) {
-    const v5andv6metrics = auditRefs.filter(audit => audit.group === 'metrics');
-    const fci = auditRefs.find(audit => audit.id === 'first-cpu-idle');
-    const fmp = auditRefs.find(audit => audit.id === 'first-meaningful-paint');
-    if (fci) v5andv6metrics.push(fci);
-    if (fmp) v5andv6metrics.push(fmp);
+    const metrics = auditRefs.filter(audit => audit.acronym);
```
Comment (patrickhulce) [inline lighthouse-core/report/html/renderer/performance-category-renderer.js:115]: this feels a little less clear and AFAICT no audits changed groups in this PR. why the switch to acronym?

File: lighthouse-core/config/default-config.js [hunk 2]
```diff
@@ -421,12 +421,12 @@ const defaultConfig = {
     'performance': {
       title: str_(UIStrings.performanceCategoryTitle),
       auditRefs: [
-        {id: 'first-contentful-paint', weight: 15, group: 'metrics', acronym: 'FCP', relevantAudits: m2a.fcpRelevantAudits},
-        {id: 'speed-index', weight: 15, group: 'metrics', acronym: 'SI'},
+        {id: 'first-contentful-paint', weight: 10, group: 'metrics', acronym: 'FCP', relevantAudits: m2a.fcpRelevantAudits},
```
Comment (patrickhulce) [inline lighthouse-core/config/default-config.js:424]: I totally missed this `m2a` business, sounds like someone has some apologizing to do to `atob`/`btoa` ;)

**Review by patrickhulce — COMMENTED** [review]
weights are beautiful 👌

File: lighthouse-core/report/html/renderer/performance-category-renderer.js (hunk 1 above)
Comment (paulirish) [reply lighthouse-core/report/html/renderer/performance-category-renderer.js]: yeah fair.  i reverted the code changes and left a todo.

the code's a little awkward as acronym was just added, but we want to support old LHRs that don't have it. (and we want to get FMP/FCPUI numbers too, despite them being hidden).

anyway, should be better?

**Review by patrickhulce — APPROVED** [review]
LGTM 🎉 

🚢  it!

**Review by brendankenny — APPROVED** [review]
LGTM

**Pull request merged**